### DISCUSSION TITLE
How to secure publicly exposed admin’s site

### DISCUSSION THREAD

**Discussion by tk0485** [keycloak/keycloak#21534] [Q&A] [answered]
I have a keycloak deployed using the bitnami helm chart. I have keycloak exposed to the internet because I have external services that needs to access it. I’m trying to harden keycloak by disabling the admin api and admin features but I still need the admin api for internal use. 
I thought of deploying a secondary instance of keycloak that is publicly exposed with admin api disabled and another internal deployment with the api enabled and both instances share the same cache and database. 
is this the best way for my usecase? Anyone has a similar setup or a different solution?

**Comment by christian-2** [comment, marked as answer]
Duplicate of #12481.
